### PR TITLE
HTML portion to README

### PR DESCRIPTION
**Pull request by christophbrosch**
Forgot to add the HTML portion to the README.

Added an automatic creation of the <head> tag if it's missing in the template.
Ran into this issue while setting everything up on my local machine.

### CODE REVIEW

**Comment by christophbrosch** [comment]
Noticed that @click should actually work if im not mistaken. But it doesn't in the example

**Comment by maximdeclercq** [comment]
> Noticed that @click should actually work if im not mistaken. But it doesn't in the example

BeautifulSoup breaks/removes invalid attributes (starting with @), which is something I'll have to look into. You can use `v-on:` in the meantime. I have made an issue: #7.

File: README.md [hunk 1]
```diff
@@ -45,9 +46,25 @@ class Home(DjangoVueComponent):
     vue_data = {'count': 0}
 ```
 
-## Roadmap
+```HTML
+{% /templates/components/counter.html %}
```
Comment (maximdeclercq) [inline README.md]: Is this supposed to be a comment or a tag? I think you meant `{% comment ... %}` or something like `<!-- ... -->`? Very nice additions so far!

File: django_vue/views.py [hunk 2]
```diff
@@ -77,8 +77,12 @@ def get(self, request: HttpRequest, *args, **kwargs):
         response.render()
         soup = BeautifulSoup(response.content, "lxml")
 
+        # Add empty head tag in case it is missing in the template
+        if soup.find("head") == None:
```
Comment (maximdeclercq) [inline django_vue/views.py]: Please refer to #7 for a possible cleaner solution ;).

**Comment by christophbrosch** [comment]
closed this by accident, didn't know forcing my master fork to be your master branch would automatically close an issue related to the changed I made. But can remain closed for now.

**Comment by maximdeclercq** [comment]
You can always merge my master into your master to sync up (these merge commits normally wouldn't show up in a pull request I think). I also advise you to make some branches on your fork so you don't lose any commits when forcing your master to mine :). 
Anyway, I actually really liked your original pull request! It was a very good draft and actually helped people to try this repo out who haven't read my Reddit post.

**Pull request closed without merging**